CONTRIBUTING.md

```diff
@@ -26,133 +26,169 @@ We specifically do not merge PRs **without prior issues** that:
 
 ---
 
-## Core Team
+### Building OpenTofu
 
-The Core Team consists of the individuals in the [MAINTAINERS](MAINTAINERS) file. This team exists as stewards of OpenTofu: to triage issues, to implement features, to help with and review community contributions, and to communicate with the Technical Steering Committee.
+There are two ways to build OpenTofu: natively and in a container. Building natively means you are building directly where your code is located without dynamically creating a container. In contrast, building in a container means you are using a build container to run the build process only.
 
-### Issue Triaging
+#### Building natively
 
-As issues are filed in the OpenTofu project, they go through a processes driven by the Core Team.  This process is in place to prevent duplicate work and to ensure that a discussion happens before work is contributed to avoid frustration.
+To build OpenTofu natively, you will need to install Go in the environment you are running in. You can then run the `go build` command from your OpenTofu source directory as follows:
 
-Steps:
-* Issue is filed and given the `pending-decision` label and an additional label to identify their type (`bug`, `enhancement`, `rfc`).
-* Issue will first be discussed between the Core Team and the community to iron out any missing details.
-* Once the Issue is well understood, the Core Team may decide to accept it, reject it, or pass the decision along to the [Technical Steering Committee](https://github.com/opentofu/opentofu/blob/main/TSC_SUMMARY.md) (`pending-steering-committee-decision`).
-  - Occasionally, the Core Team may wait to make a decision to gauge the level of community interest and will add the label `needs-community-input`.
-  - To advocate for an issue, give it a reaction and/or add a comment.
-* If Accepted:
-  - It will have the `pending-decision` label removed and the `accepted` label added.
-  - The Core Team may assign one of their members to work on it or may wait for a community member to ask for it to be assigned to them.
-  - It may sometimes be labeled with `help-wanted` or `good-first-issue` when the Core Team hopes that someone in the community will be able to pitch in and help on it.
-* If Rejected:
-  - Not all issues will make it into OpenTofu, but the decision process should be clear and documented.
-  - The issue will be closed.
+```sh
+GOOS=linux GOARCH=amd64 go build -o tofu -v -buildvcs=false ./cmd/tofu
+```
 
+This command will produce a `tofu` binary in your current directory, which you can test by running `./tofu --version`.
 
----
+> [!TIP]
+> Replace the `GOOS` and `GOARCH` values with your target platform if you wish to cross-compile.
```

Comment:
might be a good idea to link to the docs on this (https://pkg.go.dev/cmd/go#hdr-Compile_and_run_Go_program)